app/views/custom_fields/_form.html.erb

```diff
@@ -62,6 +59,17 @@ See COPYRIGHT and LICENSE files for more details.
                  }
     %>
   </div>
+  <div class="form--field-extra-actions" <%= format_dependent.attributes(:enterpriseBanner) %>>
+    <%= angular_component_tag "opce-enterprise-banner",
+                              inputs: {
+                                collapsible: true,
+                                opReferrer: "custom-field-hierarchy",
+                                textMessage: "PLACEHOLDER - upgrade to Starkiller Base",
```

Comment:
Hi @Kharonus, here is still a placeholder UI copy in the PR.

Reply:
Yes, there is a [TASK](https://community.openproject.org/wp/59219) for Maya, to provide the correct text.

My take was, as this is behind a feature flag, we can merge it even with the placeholder. But before removing the flag, we need to replace it.